client/customer-effort-score-tracks/customer-effort-score-tracks-container.js

```diff
@@ -48,7 +50,7 @@ function CustomerEffortScoreTracksContainer( {
 					key={ index }
 					action={ item.action }
 					label={ item.label }
-					onSubmitLabel={ item.onsubmit_label }
+					onSubmitLabel={ item.onSubmitLabel }
```

Comment:
This is actually meant to be `onsubmit_label` as it is passed in from the PHP side of things (https://github.com/woocommerce/woocommerce-admin/blob/main/src/Features/CustomerEffortScoreTracks.php#L200)

Reply:
@becdetat I was thinking about moving that logic from PHP to JS since JS also needs to pass the value. It's duplicated. What do you think about moving it to JS so that we don't have to set the value in PHP side?

Reply:
If you can move it then that would be good :-) . We still need to have a channel for generating the surveys on the server side though.

Reply:
Yup, I was thinking about removing [this line](https://github.com/woocommerce/woocommerce-admin/blob/add/5349-trigger-ces-on-filtering-analytics/src/Features/CustomerEffortScoreTracks.php#L200) only and set the default value in JS in case of undefined.

Reply:
It still needs to be settable on the server side - other consumers (both in a8c and potentially 3PDs) will be using both the PHP and the JS (data store) approach to creating CES surveys.

Reply:
@becdetat That makes sense. I will update it to be compatible with PHP.

Reply:
Changed it back to underscore naming convention for the key in https://github.com/woocommerce/woocommerce-admin/pull/5649/commits/a2171375260fa1a0c5a6bd4f9d2fdc06fb984505